src/models/language.rs

```diff
@@ -66,6 +76,13 @@ impl PiranhaLanguage {
     self.comment_nodes().contains(&kind)
   }
 
+  pub fn should_ignore_node_for_comment(&self, node: &Node) -> bool {
```

Comment:
Documentation on this one, explaining what it does and how it's used.

Like, is the following correct?

> [Piranha] will find the nearest node to the deleted position that is not one of these. Then, if that node is a comment, it will delete it.

(Even if so, please rewrite the doc comment a bit with some more info about how this function is used to implement the algorithm above)